auraed/tests/common/mod.rs

```diff
@@ -104,9 +105,7 @@ async fn run_auraed() -> Client {
             e => break e,
         }
     }
-    .expect("failed to create client");
-
-    client
```

Comment:
was this unreachable before?

Reply:
This was fixing a clippy lint, "returning the result of a `let` binding from a block"